src/wasm/kotlin/src/arcs/Entity.kt

```diff
@@ -12,38 +12,38 @@ class StringDecoder(private var str: String) {
 
     companion object {
 
-      fun decodeDictionary(str: String): Map<String, String> {
-        val decoder = StringDecoder(str)
-        val dict = mutableMapOf<String, String>()
+        fun decodeDictionary(str: String): Map<String, String> {
```

Comment:
After looking around github.com/google and a bunch of other libraries, it seems that 4-spaces is the way to go afterall.  I'm sorry for the churn on this. 

It might make dealing lint on g3 a pain if people try to modify Arcs kotlin over there in order to contribute back to Github, but I think it's probably for the best to keep Arcs Github in line with how google tends to write open source kotlin.